backend/app/schemas/llm.py

```
@@ -0,0 +1,78 @@
+from pydantic import BaseModel
+from typing import List, Optional, Dict, Union
+
+class LLMMessage(BaseModel):
+    role: str
+    content: str
+
+class CompletionRequest(BaseModel):
+    model: str | None = "gpt-3.5-turbo"
+    messages: List[LLMMessage]
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)